torch/csrc/jit/tensorexpr/ir.h

```diff
@@ -239,7 +239,7 @@ class Max : public BinaryOpNode<Max> {
   bool propagate_nans_;
 
  public:
-  Max(const Expr* lhs, const Expr* rhs, bool propagate_nans)
+  Max(const Expr* lhs, const Expr* rhs, bool propagate_nans = false)
```

Comment:
Why do we need this change?

Reply:
it's a legacy change from previous commits, removed now.